Formula/g/gdb.rb

```diff
@@ -62,6 +61,13 @@ def install
       --disable-binutils
     ]
 
+    # Fix: Apple Silicon build, this is only way to build native GDB
+    if OS.mac? && Hardware::CPU.arm?
+      # Workaround: "--target" must be "faked"
+      args << "--target=x86_64-apple-darwin20"
+      args << "--program-prefix="
+    end
```

Comment:
Since this is the only way to get an Apple Silicon build of GDB, perhaps this could/should be handled upstream?

Reply:
If you can fix this upstream sure

Reply:
gdb is a very slowly developing project, my simple bugs reported a few years ago are still not fixed, I can give you a quite long list...

Reply:
I have 5 year old simple bugs filed against GDB that are still 'unconfirmed' This'll never get fixed upstream.

Reply:
I'm inclined to merge this as is based on the speed of upstream responses. If someone wants to submit a patch or something, we can utilize that moving forward.